code/modules/research/designs/design_types.dm

```diff
@@ -6,1430 +145,893 @@
-	sort_string = "VAGAB"
 	price = 500
 
+/datum/design/item/tool/pneum_crow
+	name = "pneumatic crowbar"
+	desc = "When you realy need to crack open something."
+	id = "pneum_crow"
+	materials = list(MATERIAL_STEEL = 3200, MATERIAL_PLASTIC = 600, MATERIAL_SILVER = 50)
+	build_type = PROTOLATHE
+	build_path = /obj/item/weapon/tool/crowbar/pneumatic
+
+/datum/design/item/tool/combi_driver
+	name = "combi driver"
+	desc = "Drive screws, drive bolts, drill bones, you can do everything with it."
+	id = "combi_driver"
+	materials = list(MATERIAL_STEEL = 1600, MATERIAL_PLASTIC = 1000, MATERIAL_SILVER = 50)
+	build_type = PROTOLATHE
+	build_path = /obj/item/weapon/tool/screwdriver/combi_driver
+
 /datum/design/item/tool/airlock_brace
 	name = "airlock brace"
 	desc = "Special door attachment that can be used to provide extra security."
 	id = "brace"
-	req_tech = list(TECH_ENGINEERING = 3, TECH_MATERIAL = 2)
 	materials = list(MATERIAL_STEEL = 2000, MATERIAL_GLASS = 50)
 	build_path = /obj/item/weapon/airlock_brace
-	sort_string = "VAGAC"
 
 /datum/design/item/tool/brace_jack
 	name = "maintenance jack"
 	desc = "A special maintenance tool that can be used to remove airlock braces."
 	id = "bracejack"
-	req_tech = list(TECH_ENGINEERING = 3, TECH_MATERIAL = 2)
 	materials = list(MATERIAL_STEEL = 120)
 	build_path = /obj/item/weapon/tool/crowbar/brace_jack
-	sort_string = "VAGAD"
 	price = 1500
 
 /datum/design/item/tool/clamp
 	name = "stasis clamp"
 	desc = "A magnetic clamp which can halt the flow of gas in a pipe, via a localised stasis field."
 	id = "stasis_clamp"
-	req_tech = list(TECH_ENGINEERING = 4, TECH_MAGNET = 4)
 	materials = list(MATERIAL_STEEL = 500, MATERIAL_GLASS = 500)
 	build_path = /obj/item/clamp
-	sort_string = "VAGAE"
 	price = 600
 
 /datum/design/item/tool/price_scanner
 	name = "price scanner"
 	desc = "Using an up-to-date database of various costs and prices, this device estimates the market price of an item up to 0.001% accuracy."
 	id = "price_scanner"
-	req_tech = list(TECH_MATERIAL = 2, TECH_MAGNET = 1) // Why was this as high as it was anyway?
 	materials = list(MATERIAL_STEEL = 3000, MATERIAL_GLASS = 3000, MATERIAL_SILVER = 250)
 	build_path = /obj/item/device/price_scanner
-	sort_string = "VAGAF"
 	price = 750
 
 /datum/design/item/tool/advanced_welder
 	name = "advanced welding tool"
 	desc = "This welding tool feels heavier in your possession than is normal."
 	id = "experimental_welder"
-	req_tech = list(TECH_ENGINEERING = 5, TECH_PHORON = 4)
 	materials = list(MATERIAL_STEEL = 120, MATERIAL_GLASS = 50)
 	build_path = /obj/item/weapon/tool/weldingtool/advanced
-	sort_string = "VAGAG"
 	price = 250
 
 /datum/design/item/tool/oxycandle
 	name = "oxycandle"
 	desc = "a device which, via a chemical reaction, can pressurise small areas."
-	id="oxycandle"
-	req_tech = list(TECH_ENGINEERING = 2)
-	materials = list(MATERIAL_STEEL = 3000)
-	chemicals = list(/datum/reagent/sodiumchloride = 20, /datum/reagent/acetone = 20)
```

Comment:
the removal of chemical functionality from the protolathe seems like a significant step backwards. This was a cool feature